docs/source/releases/v2.0.rst

```diff
@@ -47,6 +47,9 @@ What's new in Oscar 2.0?
 - Added a ``filter_by_attributes`` method on the ``ProductQuerySet``,
   to allow database-level filtering of products by attribute.
 
+- Added images re-order to the dashboard product edit page, implement extra empty form addition to the images formset
```

Comment:
Grammar - suggest "Added support for re-ordering product images in the dashboard, and for adding an arbitrary number of additional images into the formset."

Reply:
Fixed.